### ISSUE TITLE
Unexpected focus outline on b-form-group description

### ISSUE THREAD

**Issue by mikkokut** [bootstrap-vue/bootstrap-vue#5302]
### Describe the bug
`b-form-group` description element shows focus outline when clicked.

### Steps to reproduce the bug

1. Go to https://bootstrap-vue.org/docs/components/form-group
2. Click on "Let us know your name."
4. See the focus outline

### Expected behavior
Should not show the focus outline as that should have been removed by Bootstrap _reboot.scss: https://github.com/twbs/bootstrap/blob/v4.4.1/scss/_reboot.scss#L66.

### Versions

**Libraries:**

- BootstrapVue: 2.13.0
- Bootstrap: 4.4.1

**Environment:**

- Device: Mac
- OS: macOS Catalina
- Browser: Chrome
- Version: 81.0.4044.129

### Demo link
https://bootstrap-vue.org/docs/components/form-group

**Comment by Hiws** [comment]
The way bootstrap handles outlines changed from version `v4.3.1` to `v4.4.0`.
In `v4.3.1` they always removed the outline if the element had a tabindex of `-1`, but this was changed in `v.4.4.0` to let the browser handle whether or not it should show (once they actually implement it)

*v4.3.1*
https://github.com/twbs/bootstrap/blob/8fa0d3010112dca5dd6dd501173415856001ba8b/scss/_reboot.scss#L62-L64
*v.4.4.0*
https://github.com/twbs/bootstrap/blob/593574d510050dc2d5eeafe70a58ae182f020480/scss/_reboot.scss#L66-L68

The selector they're using in `4.4.0` isn't actually implemented yet in any browsers.
See [CanIUse](https://caniuse.com/#search=%3Afocus-visible), which is why it's not doing anything.

I believe the idea is that once it's implemented, the focus will be hidden from users who focused the element with a mouse. But those who focused it with a keyboard or similar would see the outline.

There exists a polyfill too, that people can use until browsers implement it natively https://github.com/WICG/focus-visible - I have not used this polyfill so i dont' know how well it functions.

(this was more about why it's happening instead of a solution)

**Comment by tmorehouse** [comment]
Note the `master` branch at twbs/bootstrap is for to upcoming v5 of Bootstrap.  You need to check the `v4-dev` branch: https://github.com/twbs/bootstrap/blob/v4-dev/scss/_reboot.scss

**Comment by mikkokut** [comment]
I updated the link to point the v4.4.1 tag. I get the same result in my app too but the weird thing is that it works in dev build but not in production build. I use nuxt for my app.

**Comment by tmorehouse** [comment]
Strange... as dev/prod should use the same SCSS reboot.

You can manually create an override CSS to set `outline` to `0` to hide the focus outline.... but it is _highly recommended_ to not do so for accessibility reasons.  The outline currently only appears if users click the feedback/description element(s).

In the majority of user cases, users will not see a focus ring unless they explicitly click the feedback/description text.  The `tabindex="-1"` is required for proper accessibility for screen reader users to focus the description or feedback text to re-read the content.

**Comment by mikkokut** [comment]
I created sample project for reproducing the issue.
https://github.com/mikkokut/nuxt-bootstrap-outline-bug

Prepare
```
git clone https://github.com/mikkokut/nuxt-bootstrap-outline-bug.git
cd nuxt-bootstrap-outline-bug
yarn install
```

Does not appear in dev build
```
yarn dev
```

Appears in production build
```
yarn build && yarn start
```

**Comment by tmorehouse** [comment]
This might be a CSS minimizer bug that is fiddling with the CSS minification (CSS minification happens in production mode only).

Nuxt uses css-nano by default, which has a few bugs that might affect this issue.

**Comment by Soviut** [comment]
I ran into this issue and it turned out I'd used the Bootstrap styling guide, so I only had the bootstrap.scss imported into my main scss file. What I needed was:

```
@import '~bootstrap/scss/bootstrap.scss';
@import '~bootstrap-vue/src/index.scss';
```

Which is documented here https://bootstrap-vue.org/docs/reference/theming#default-theme-colors

**Comment by jackmu95** [comment]
Chrome v86 adds support for the new `:focus-visible` pseudo selector and the focus ring should be gone.
